discord-impl/src/main/java/net/starype/quiz/discordimpl/game/DiscordGameServer.java

```diff
@@ -88,7 +91,12 @@ public void onPlayerScoreUpdated(Player<?> player) {
 
     @Override
     public void onQuestionReleased(Question question) {
-        sendAsLatexFile(question.getRawQuestion());
+        String rawQuestion = question.getRawQuestion();
+        if(question.isTagAttached("Math")) {

```

Comment:
I don't believe that the formatting of the answer as `LaTeX` or `Default` should be decide on whether or not the question possesses a given tag ! We could either add a new properties in the `TOML` or detect certain characteristic in the text (such as `"\text",```java ` ). We could also add a keyword in the answer that decide the type of formatting (such as if answer starts with `"LaTeX"` format as LaTeX) or even better parse only certain area as so.

Reply:
Thought it is a lot of work and could be achieve in another PR. Yet, I don't think we should merge before fixing that. But it is possible to merge first if you just want the code as fast as possible on the `master`

Reply:
I agree, let's make another PR for this. We will wait before merging this one